platforms/openid-connect/README.md

```diff
@@ -0,0 +1,110 @@
+## Unleash Community Docker Image with OpenID Connect Support
+
+**Useful links:**
+
+- [Docker image on dockerhub](https://hub.docker.com/r/unleashorg/unleash-community-openid-connect)
+- [passport-openid-connect](https://www.passportjs.org/packages/passport-openidconnect/)
+
+For an example usage see the example below.
+You can also visit the [passport](https://www.passportjs.org/docs/) and [openid-connect](https://www.passportjs.org/packages/passport-openidconnect/) documentations for further information.
+
+## Example
+
+This shows the usage of the OpenIdConnectStrategy with a keycloak realm as IDP.
+It authenticates the user and checks some custom roles to give the user some initial permissions. 
+It also updates the role in case it changed, so you can completely control unleash permissions through roles in keycloak.
+You need to configure your client accordingly, so it returns roles as a `resource_access` object.
+
+If you only need the authentication, just remove that part.
+
+```javascript
+const unleash = require('unleash-server')
+
+function openIdConnect(app, config, services) {
+    const {userService} = services;
+
+    passport.use(new OpenIdConnectStrategy({
+        issuer: `${host}/realms/${realm}`,
+        authorizationURL: `${host}/realms/${realm}/protocol/openid-connect/auth`,
```

Comment:
Would be nice if you were to use env variables here. Please consider adding these: 

```javascript
const authHost = process.env.UNLEASH_AUTH_HOST;
const realm = process.env.UNLEASH_AUTH_REALM;
const clientID = process.env.UNLEASH_AUTH_CLIENT_ID;
const clientSecret = process.env.UNLEASH_AUTH_CLIENT_SECRET;
const host = process.env.UNLEASH_HOST;

// Then these can be used within the strategy options like so:
{
    clientID,
    clientSecret,
    issuer: `${authHost}/realms/${realm}`,
    callbackURL: `${host}${context}/callback`,
    authorizationURL: `${authHost}/realms/${realm}/protocol/openid-connect/auth`,
    tokenURL: `${authHost}/realms/${realm}/protocol/openid-connect/token`,
    userInfoURL: `${authHost}/realms/${realm}/protocol/openid-connect/userinfo`,
}
```

to be somewhat consistent with the azure docker image:
https://github.com/Unleash/unleash-examples/blob/main/v4/securing-azure-auth/azure-auth-hook.js